packages/okta-react/README.md

```diff
@@ -179,30 +179,21 @@ Security is the top-most component of okta-react. This is where most of the conf
 
 #### Configuration options
 
+These options are used by `Security` to configure the [Auth](https://github.com/okta/okta-oidc-js/blob/master/packages/okta-react/src/Auth.js) object. The most commonly used options are shown here. See [Configuration Reference](https://github.com/okta/okta-auth-js#configuration-reference) for an extended set of supported options.
+
 - **issuer** (required) - The OpenId Connect `issuer`
-- **client_id** (required) - The OpenId Connect `client_id`
-- **redirect_uri** (required) - Where the callback handler is hosted
+- **clientId** (required) - The OpenId Connect `client_id`
+- **redirectUri** (required) - Where the callback handler is hosted
 - **scope** *(optional)* - Reserved or custom claims to be returned in the tokens. Default: `['openid', 'email', 'profile']`
-- **response_type** *(optional)* - Desired token types. Default: `['id_token', 'token']`
-- **grantType** *(optional)* - Can be `implicit` (default) or `authorization_code` (for PKCE flow)
```

Comment:
Was **grantType** ever published for okta-react, or is this leftover from earlier work in this same release?

Reply:
its left over, we have not released pkce support for react yet